documentation/custom-image-support/README.md

```diff
@@ -0,0 +1,78 @@
+# ADE Custom Image Support
+This page is designed to help customers understand how to build and utilize custom images within their environment definitions for deployments in Azure Deployment Environments. 
+
+To see how the ADE-authored Core and ARM/Bicep images are structured, check out the Runner-Images folder [here](../../Runner-Images/README.md).
+
+In order to use the ADE CLI, you will need to use an ADE-authored image as a base image. More information about the ADE CLI can be found [here](./ade-cli-docs/README.md).
+
+## Creating and Building the Docker Image
+
+### FROM Statement
+
+If you are wanting to build a Docker image to utilize ADE deployments and access the ADE CLI, you will want to base your image off of one of the ADE-authored images. This can be done by including a FROM statement within a created DockerFile for your new image pointing to an ADE-authored image hosted on Microsoft Artifact Registry.
+
+Here's an example of that FROM statement, pointing to the ADE-authored core image:
+```docker
+FROM mcr.microsoft.com/deployment-environments/runners/core:latest
+```
+
+This statement pulls the most recently-published core image, and makes it a basis for your custom image.
+
+### RUN Statement
+
+Next, you can use the RUN statement to install any additional packages you would need to use within your image. ADE-authored images are based off of the Azure CLI image, and have the ADE CLI and JQ packages pre-installed. You can learn more about the Azure CLI [here](https://learn.microsoft.com/en-us/cli/azure/), and the JQ package [here](https://devdocs.io/jq/).
+
+Here's an example used in our Bicep image, installing the Bicep package within our Dockerfile.
+```docker
+RUN az bicep install
+```
+
+### Executing Operation Shell Scripts
+
+Within the ADE-authored images, operations are determined and executed based off of the operation name. Currently, the two operation names supported are 'deploy' and 'delete', with plans to expand this moving forward.
+
+To set up your custom image to utilize this structure, specify a folder at the level of your Dockerfile named 'actions.d', and specify two files, 'deploy.sh', and 'delete.sh'. The 'deploy' shell script will run when your environment is created or redeployed, and the 'delete' shell script will run when your environment is deleted. You can see examples of this within this repository under the Runner-Images folder for the ARM-Bicep image.
```

Comment:
Prior to publishing, we should remove references to "actions". I'll submit a PR changing the folder to something like "scripts".

Reply:
updated to reflect the new Dockerfiles referencing 'scripts' instead of 'actions.d'